fortitude/src/check.rs

```diff
@@ -988,8 +917,10 @@ pub fn check(args: CheckArgs, global_options: &GlobalConfigArgs) -> Result<ExitC
     let ast_entrypoints = ast_entrypoint_map(&rules);
 
     let start = Instant::now();
+
     let files = get_files(
         files,
+        project_root(path_absolutize::path_dedot::CWD.as_path())?,
```

Comment:
I guess this should actually be the directory containing the parent `fortitude.toml`/`fpm.toml` file? We do search up the directory structure to find one, so we should probably use that here

Reply:
Ignore that, I was too focussed on the `CWD` bit I missed this was a function call!